dt-core/admin/components/setup-wizard-keys.js

```diff
@@ -0,0 +1,209 @@
+import {
+  html,
+  css,
+} from 'https://cdn.jsdelivr.net/gh/lit/dist@2.4.0/all/lit-all.min.js';
+import { OpenLitElement } from './setup-wizard-open-element.js';
+
+export class SetupWizardKeys extends OpenLitElement {
+  static styles = [
+    css`
+      :host {
+        display: block;
+      }
+    `,
+  ];
+
+  static get properties() {
+    return {
+      step: { type: Object },
+      firstStep: { type: Boolean },
+      _options: { type: Object },
+      saving: { type: Boolean },
+    };
+  }
+
+  back() {
+    this.dispatchEvent(new CustomEvent('back'));
+  }
+  async next() {
+    this.saving = true;
+    await window.dt_admin_shared.update_dt_options(this._options);
+    this.saving = false;
+
+    this.dispatchEvent(new CustomEvent('next'));
+  }
+
+  constructor() {
+    super();
+    this.saving = false;
+    this._options = {
+      dt_mapbox_api_key: '',
+      dt_google_map_key: '',
+    };
+    this.show_mapbox_instructions = false;
+    this.show_google_instructions = false;
+  }
+
+  render() {
+    this._options.dt_mapbox_api_key = this.step.config.dt_mapbox_api_key;
+    this._options.dt_google_map_key = this.step.config.dt_google_map_key;
```

Comment:
Does this mean the instructions will be hidden?
Are they hiding for dev purposes or on purpose?

Reply:
hidden and a button to show them

Reply:
ah nice 🎉